app/lib/pre_assembly/digital_object.rb

```diff
@@ -141,18 +141,22 @@ def openable?
       Dor::Services::Client.object(druid.druid).version.openable?
```

Comment:
seems like you could do the same here too?

Reply:
Thanks. Accepted.